.circleci/config.yml

```diff
@@ -45,7 +45,7 @@ commands:
       - run:
           name: Run semantic release in dry-run mode
           command: |
-            npx -p @semantic-release/git@10.0.1 -p semantic-release@21.0.2 semantic-release --branches ${CIRCLE_BRANCH} --dry-run > sem-rel-dry-run.txt
```

Comment:
> Warning
> You likely do not need this plugin to accomplish your goals with semantic-release.
Please consider our [recommendation against making commits during your release](https://semantic-release.gitbook.io/semantic-release/support/faq#making-commits-during-the-release-process-adds-significant-complexity) to avoid unnecessary headaches.

From https://github.com/semantic-release/git